docs/content/guides/authn/oidc_first_party_auth.adoc

```diff
@@ -13,11 +13,11 @@ description: This guide will walk you through the process of integrating heimdal
 
 By the end of this guide, you'll have a functional setup where heimdall uses https://www.keycloak.org/[Keycloak] to authenticate users and route requests based on their authentication status and roles for role-based access control.
 
-Although this guide uses Keycloak as identity provider (IDP), you can achieve the same results with https://zitadel.com[Zitadel], https://github.com/malach-it/boruta-server[Boruta], or any other OpenID Connect-compatible IDP.
+Although this guide uses Keycloak as identity provider (IDP), you can achieve the same results with https://zitadel.com[Zitadel], https://github.com/malach-it/boruta-server[Boruta], or any other OpenID Connect compatible IDP.
 
 == Overview
 
-In this guide, we'll set up a Docker Compose environment where heimdall secures  services and controls access to specific endpoints:
+In this guide, we'll set up a Docker Compose environment where heimdall secures services to control access for specific endpoints:
```

Comment:
The new sentence could be misinterpreted, as "to control access for specific endpoints" makes it sound like securing services is done for the purpose of controlling access, rather than being two related but distinct actions. I find the old sentence to be clearer and more natural.